pkg/manifests/common_test.go

```diff
@@ -19,11 +19,11 @@ var (
 		Name   string
 		Config *config.Config
 	}{
-		{Name: "namespace", Config: &config.Config{
+		{Name: "Namespace", Config: &config.Config{
 			NS: "test-namespace",
 		}},
 		{
-			Name: "another-namespace",
+			Name: "another-Namespace",
```

Comment:
You replace all this?

Reply:
not sure how/why I made this but getting rid of this